apps/council-frontend/src/ui/delegate/DelegatesList/DelegateProfileRow.tsx

```diff
@@ -41,126 +47,143 @@ function DelegateProfileRow(props: DelegateProfileRowProps): ReactElement {
     delegate.commonwealthName ||
     delegate.name ||
     formatWalletAddress(delegate.address);
+
   const delegateNameElement = (
     <span className="truncate">{formattedDelegateName}</span>
   );
+
+  const showProfileRow =
+    !delegateListFilter ||
+    filter(
+      [ensName?.toLowerCase() || "", delegate.address.toLowerCase()],
+      delegateListFilter,
+    ).length > 0;
+
+  if (!showProfileRow) {
```

Comment:
Then this becomes:

```
if (!renderPredicate(delegate.address, allDelegates) {
   return null;
}
```

And the fuzzaldrin stuff goes in the parent component that's rendering the profile rows.